charts/evm-bridge-withdrawer/values.yaml

```diff
@@ -27,6 +27,7 @@ config:
   minExpectedFeeAssetBalance: "1000000"
   rollupAssetDenom: ""
   evmContractAddress: "0x"
+  evmContractIsERC20: "true"
```

Comment:
I set the default to true here because it's a little safer. If it is not an ERC20, it will fail on startup. Otherwise it could startup and not read the decimals resulting in using the wrong decimals.